tindergames/src/test/java/org/teamcifo/logic/LogInTest.java

```diff
@@ -0,0 +1,103 @@
+package org.teamcifo.logic;
+
+import static org.junit.jupiter.api.Assertions.*;
+import com.github.javafaker.Faker;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.teamcifo.domain.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LogInTest {
+
+    private UserManager userManager;
+    private LogIn logIn;
+    private List<User> users;
+
+    @BeforeEach
+    void setUp() {
+        userManager = new UserManager();
```

Comment:
Same here, don't create an object, simply invoke the `UserManager` static methods

Reply:
Same as before